packages/analytics/shippers/fullstory/src/fullstory_shipper.ts

```diff
@@ -20,6 +20,11 @@ import { loadSnippet } from './load_snippet';
 
 export type FullStoryShipperConfig = FullStorySnippetConfig;
 
+/**
+ * FullStory's custom events rate limit is very aggressive. We'll only send some allowed events.
+ */
+const CUSTOM_EVENT_TYPES_ALLOWLIST = ['Loaded Kibana'];
```

Comment:
For extensibility (and testability) purposes, should we allow configuring this list by adding a property in `FullStoryShipperConfig`?. Property could be optional and defaulting to this hardcoded list.

Reply:
I love this idea! I'll apply that!